docker/unitTest.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
You should run the actions in unitTest.sh in ciUnitTest.yml instead, this is so that it'll be easier to reuse the docker image when your build configuration changes.